### PR TITLE
Centrality reco modification for 2018 heavy ion run 

### PR DESCRIPTION
**Pull request by YeonjuGo**
This is a backport of PR #24892

### CODE REVIEW

**Comment by perrotta** [comment]
backport of #24892

**Comment by perrotta** [comment]
please test

**Comment by slava77** [comment]
@cmsbuild please test

**Comment by slava77** [comment]
+1

for https://github.com/cms-sw/cmssw/pull/24889	16baa76
- the backport from master is trivial: the same topic branch is used
- jenkins tests pass and comparisons with the baseline show differences only in the HI workflows in the centrality monitoring plots

**Comment by fabiocos** [comment]
@slava77 @perrotta this PR affects DataFormats, so the sooner it enter the better it is (although it is an extension with new members). Anyway this cannot be for a fast patch to try again a replay at T0, it must be a full 10_3_1

**Comment by BetterWang** [comment]
Hi @fabiocos , is it possible to make 10_3_1 release soon? Do we have to report to anyone?@mandrenguyen can also comment.

**Comment by perrotta** [comment]
@cmsbuild please test

**Comment by perrotta** [comment]
@YeonjuGo @mandrenguyen : shouldn't this turning off of pixeltracks for XeXe also get integrated in the master?

**Comment by mandrenguyen** [comment]
@YeonjuGo Please update the PR to the master branch as well

**Comment by fabiocos** [comment]
@BetterWang @YeonjuGo @mandrenguyen please notice that the master version of this PR is causing a crash in the IB, see #24981 , could someone urgently look at it? We cannot backport things that are failing...

**Comment by fabiocos** [comment]
please test workflow 148.0

**Comment by YeonjuGo** [comment]
@perrotta @mandrenguyen I made a new PR #24994 to have the same commit to master with the last commit of this PR since I can't update the PR #24892 because it's already merged.

**Comment by perrotta** [comment]
please test workflow 148.0

File: Configuration/StandardSequences/python/Reconstruction_cff.py [hunk 1]
```diff
@@ -163,7 +163,7 @@
 from Configuration.Eras.Modifier_pp_on_AA_2018_cff import pp_on_AA_2018
 from RecoHI.HiTracking.HILowPtConformalPixelTracks_cfi import *
 _highlevelreco_HI_wPixTracks = highlevelreco.copy()
-pp_on_AA_2018.toReplaceWith(highlevelreco, cms.Sequence(_highlevelreco_HI_wPixTracks* hiConformalPixelTracksSequencePhase1))
+(pp_on_XeXe_2017 | pp_on_AA_2018).toReplaceWith(highlevelreco, cms.Sequence(_highlevelreco_HI_wPixTracks* hiConformalPixelTracksSequencePhase1))
```
Comment (slava77) [inline Configuration/StandardSequences/python/Reconstruction_cff.py]: please reshuffle the hiConformalPixelTracksSequencePhase1 addition: move the import of HILowPtConformalPixelTracks as well as the addition of hiConformalPixelTracksSequencePhase1 above before the addition of the centrality.

Comment (YeonjuGo) [reply Configuration/StandardSequences/python/Reconstruction_cff.py]: @slava77 thanks a lot. I just made a change

**Comment by fabiocos** [comment]
please test

**Comment by fabiocos** [comment]
@slava77 @perrotta as far as I can see that the backport of #25011 is ok. Could you please check and sign it in case? Unless you see issues I would like to merge this PR by today.

Strictly speaking this is a modification of the event content (addition of data members), but a known late feature to be imported before the real production use starts.

**Comment by fabiocos** [comment]
+operations

the update of StandardSequences is the backport of #25011 merged in master

**Comment by slava77** [comment]
+1

for https://github.com/cms-sw/cmssw/pull/24889	3356636
- this PR is now in sync with follow-up updates in the master: the original #24892, an update to fix issues at run time in XeXe in #25000, and an update to fix issues running in scheduled mode (test express configuration is using scheduled mode) in #25011 
- jenkins tests pass and comparisons with the baseline show differences only in the HI workflows in the centrality variable

**Comment by fabiocos** [comment]
+1

@slava77 thank you

**Pull request merged**